shared/profile/user/index.js

```diff
@@ -350,13 +344,20 @@ class User extends React.Component<Props, State> {
     const {itemsInARow, itemWidth} = widthToDimentions(this.state.width)
     // TODO memoize?
     let chunks = this.state.width ? chunk(friends, itemsInARow) : []
-    if (chunks.length === 0 && this.props.followingCount !== null && this.props.followingCount !== null) {
-      chunks.push({
-        text: this.state.selectedFollowing
-          ? `${this.props.userIsYou ? 'You are' : `${this.props.username} is`} not following anyone.`
-          : `${this.props.userIsYou ? 'You have' : `${this.props.username} has`} no followers.`,
-        type: 'noFriends',
-      })
+    if (chunks.length === 0) {
+      if (this.props.following && this.props.following) {
```

Comment:
?

Reply:
should be followers, copy and paste of the incorrect code above on line 353